engine/servers/pgvector-single-node/docker-compose.yaml

```diff
@@ -3,13 +3,17 @@ version: '3.7'
 services:
   pgvector:
     container_name: pgvector
-    image: ankane/pgvector:v0.5.1
+    image: pgvector/pgvector:pg16
     environment:
       - POSTGRES_DB=postgres
       - POSTGRES_USER=postgres
       - POSTGRES_PASSWORD=passwd
       - POSTGRES_HOST_AUTH_METHOD=trust
-      - POSTGRES_MAX_CONNECTIONS=200
+    # shared_buffers should be 25% of memory
+    # maintenance_work_mem should be ~65%
+    command: postgres -c shared_buffers=2GB -c maintenance_work_mem=5GB -c max_connections=200
+    # shm_size should be shared_buffers + maintenance_work_mem
+    shm_size: 7g
```

Comment:
These settings should be updated based on the hardware used to run the benchmark